geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -276,7 +279,7 @@ public void testScenario3() throws Exception {
 
       current_fullpath = jtaObj.currRegion.getFullPath();
       assertEquals("failed retrieving current region fullpath after txn commit",
-          "/" + DEFAULT_RGN + "/region1", current_fullpath);
+          SEPARATOR + DEFAULT_RGN + "/region1", current_fullpath);
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?